app/lib/katello/resources/cdn/katello_cdn.rb

```diff
@@ -36,12 +36,16 @@ def valid_path?(_path, _postfix)
           true
         end
 
+        def validate_cv_lce!
+          content_view_id && lifecycle_environment_id
+        end
+
         def debug_certificate
           get("/katello/api/v2/organizations/#{organization['id']}/download_debug_certificate")
         end
 
         def content_view_id
-          rs = get("/katello/api/v2/organizations/#{organization['id']}/content_views?name=#{CGI.escape(@content_view_label)}")
+          rs = get("/katello/api/v2/organizations/#{organization['id']}/content_views?search=#{CGI.escape(`label=#{@content_view_label}`)}")
```

Comment:
whats up with the back ticks. I think ruby tries to run that as an executable
```suggestion
          rs = get("/katello/api/v2/organizations/#{organization['id']}/content_views?search=#{CGI.escape('label=#{@content_view_label}')}")
```

Reply:
Hmm..React hangover..